getstarted.rst

```diff
@@ -1,77 +1,119 @@
 Getting Started
 ===============
 
+.. _requirements:
+
 Requirements
 ------------
 
 * `Python 3.4 or greater <https://www.python.org/download/>`_
 * `pip <https://pip.pypa.io/en/latest/installing.html>`_
-* virtualenv - ``pip install virtualenv``
+* `virtualenv <http://docs.python-guide.org/en/latest/dev/virtualenvs/>`_
+* `git <http://git-scm.com/download/mac>`_
+
+First, open a terminal in to the home directory and check that you have everything you need.
+
+.. code-block:: bash
+
+    python3 --version
+    pip3.4 --version
+    virutalenv --version
+    git --version
+
+If those four commands don't return anything, then follow the requirements links before continuing.
+
```

Comment:
I think that should be implied. If pip3.4 links to something besides python3.4 then I will be very confused.